wiki/Hit_object/Hyperfruit/en.md

```diff
@@ -0,0 +1,11 @@
+---
+stub: true
+---
+
+# Hyperfruit
+
+A *hyperfruit* is an effect that can be generated on a [fruit](/wiki/Hit_object/Fruit) or [drop](/wiki/Hit_object/Juice_stream#drop). This effect will be generated when the next fruit or drop can't be caught with normal dashing. This effect will also generate when the last [banana](/wiki/Hit_object/Banana) of a [spinner](/wiki/Hit_object/Spinner) can't be caught with a dash.
```

Comment:
I know little about osu!catch, but I think you should also mention hyperdash here (is it still called that?)

Reply:
I first had it mentioned but removed it since a hyperdash is the movement between a hyperfruit and its target. I already have this covered in the following PR for the catch glossary, although I've made a small change so I mention it now 👍